eks-distro-base/Makefile

```diff
@@ -288,8 +288,6 @@ $(COMPILER_BUILDER_VERSIONED_VARIANTS): BUILDCTL_BUILD_ARGS+=MINIMAL_BASE_BUILDE
 
 $(eval $(foreach ver,$(VERSIONS),$(call COMPILER_TARGET_OVERRIDES_FOR_VERSION,base-$(1)-compiler,$(ver),$(1))))
 
-$(eval $(foreach ver,$(VERSIONS),$(ALL_MINIMAL_IMAGE_TARGETS_FOR_BASE_$(call TO_UPPER,$(1))_$(ver)): VERSIONED_VARIANT=$(ver)))
```

Comment:
This was left over for compilers with runtime images. It will come back when i add the nodejs image but for golang its not needed and i need to rework it.